src/components/notifications/Notification.jsx

```diff
@@ -0,0 +1,298 @@
+            notifiId: id
+        };
+        const resp = dispatch(rejectNotifications(body));
+        resp.then((data) => {
+            data.resp.then((r) => {
+                setAccionStatus(r?.notification?.item?.status);
+                setItemIdStatus(r?.notification?.item?.item_id);
+                setProjectIdStatus(r?.notification?.item?.project_id);
+                setWatchedStatus(r?.notification?.watched);
+                setTriggeredStatus(r?.notification?.triggered);
+            })
+        })
+    }
+
+    const undoReject = (id, pId) => {
+        const body = {
+            idUser: userData.id,
+            idProject: pId,
+            notifiId: id
+        };
+        const resp = dispatch(undoRejectNotifications(body));
+        resp.then((data) => {
+            data.resp.then((r) => {
+                setAccionStatus(r?.notification?.item?.status);
+                setItemIdStatus(r?.notification?.item?.item_id);
+                setProjectIdStatus(r?.notification?.item?.project_id);
+                setWatchedStatus(r?.notification?.watched);
+                setTriggeredStatus(r?.notification?.triggered);
+            })
+        })
+    }
+    if (itemType === 'ProjectInvitation') {
+        return (
+            <ContentPrimary>
+                <ContentPoint>
+                    {triggeredStatus === false && (<NewPoint />)}
+                    {watchedStatus === false && (
+                        <ListItem style={{ backgroundColor: '#42bfad26' }}>
+                            <ProfilePictureContainer>
+                                <IconUser
+                                    user={userData}
+                                    size={56}
+                                    fontSize={24}
+                                />
+                            </ProfilePictureContainer>
+                            <ContainerNameUser>
+                                <InfoUserName gray>{date}</InfoUserName>
+                                <InfoUserName gray dangerouslySetInnerHTML={{ __html: message }} />
+                                <ContentActions>
+                                    {status === 'Proyecto Aceptado' && (
+                                        <>
+                                            {loadingNoti && (
+                                                <>
+                                                    <ActionPerformed>Proyecto aceptado</ActionPerformed>
+                                                    <LinkSeeAll href={projectUrlStatus} style={{ color: 'rgb(156 221 212)', pointerEvents: 'none' }} disabled>Ir al proyecto</LinkSeeAll>
+                                                </>
+                                            )}
+                                            {!loadingNoti && (
+                                                <>
+                                                    <ActionPerformed>Proyecto aceptado </ActionPerformed>

```

Comment:
código repetido abajo, evitarlo porfa.. si cambias solo los estilos, defínelos en otro lado y luego los usas dependiendo de `loadingNoti`

lo mismo para Proyecto Rechazado.
De hecho deberías separarlo a otro componente, recuerda siempre tu código debe ser DRY.. ese concepto está detrás de usar componentes en react 
https://javascript.plainenglish.io/5-dry-principles-to-follow-in-react-6ca0405986a9